app/src/main/res/layout/fragment_own_profile.xml

```diff
@@ -4,44 +4,109 @@
+
+            <ImageView
+                android:id="@+id/btnProfileSettings"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_alignParentEnd="true"
+                android:contentDescription="@string/content_description_profile_settings"
+                android:src="@drawable/ic_settings_button" />
+
+        </RelativeLayout>
+
+        <ImageView
+            android:id="@+id/ivProfileImage"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:contentDescription="@string/content_description_profile_image"
+            android:scaleX="0.5"
+            android:scaleY="0.5"
+            android:src="@drawable/ic_profile"
+            app:tint="@color/gray600" />
 
         <LinearLayout
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
+            android:layout_alignParentBottom="true"
             android:orientation="vertical"
             android:padding="@dimen/activity_vertical_margin">
 
             <TextView
                 android:id="@+id/tvName"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
+                android:fontFamily="@font/readexpro_semibold"
+                android:textAlignment="center"
+                android:textColor="@color/white"
                 android:textSize="@dimen/text_size_title"
                 tools:text="Name" />
 
             <TextView
                 android:id="@+id/tvUsername"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
+                android:fontFamily="@font/readexpro_regular"
+                android:textAlignment="center"
+                android:textColor="@color/gray400"
                 tools:text="username" />
 
         </LinearLayout>
-    </com.google.android.material.card.MaterialCardView>
+
+    </RelativeLayout>
+
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_weight="0.25"
+        android:background="@drawable/ic_profile_description_background"
+        android:orientation="vertical"
+        android:padding="20dp"
+        app:strokeColor="@color/black"
+        app:strokeWidth="2dp">
+
+        <TextView
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:fontFamily="@font/readexpro_semibold"
+            android:text="@string/profile_title_bio"
+            android:textColor="@color/gray600"
+            android:textSize="20sp" />
+
+        <TextView
+            android:id="@+id/tvBioDescription"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:text="Hey ho, willkommen auf meinem Profil 👋🏼" />
```

Comment:
move string to strings.xml

Reply:
It's just there as a placeholder. Going to make it a tools:text